app.yaml

```diff
@@ -0,0 +1,35 @@
+---
+application: cdash-playground
+version: 2
```

Comment:
Remember to set this to version one and a generic application name before you merge.